index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE HTML>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <title>Weather Report</title>
+    <link href="styles/index.css" rel="stylesheet" />
+</head>
+<body>
+    <div class="main-box">
+        <article id="temp-box">
+            <h3>Temperature </h3>
+                <div class="temp-display">
+                    <div class="arrows">
+                        <span id="up">⬆️</span>
+                        <span id="down">⬇️</span>
+                    </div>
+                <span id="number">55</span>
+                </div>
+        </article>
+        <article id="landscape-box">
+            <img src="assets/images/spring-rainy.png" id="landscape" /><br>
+                <input placeholder="Enter new city" name="name" id='spot'/>
+                <p id="log" >Seattle</p>   
+                <button onclick="document.getElementById('spot', 'log').value = ''">Clear </button>
```

Comment:
getElementById does not support multiple IDs. You can handle this by using `querySelectorAll("#spot", "#log")` or you could create a function in your index.js file to handle the functionality of the clear button.